ext/rotoscope/rotoscope.c

```diff
@@ -178,17 +178,17 @@ static bool in_fork(Rotoscope *config)
 
 static bool tracecmp(rs_tracepoint_t *a, rs_tracepoint_t *b)
 {
-  return (!strcmp(a->method_name, b->method_name) &&
-          !strcmp(a->entity, b->entity) &&
+  return (!strcmp(StringValueCStr(a->method_name), StringValueCStr(b->method_name)) &&
+          !strcmp(StringValueCStr(a->entity), StringValueCStr(b->entity)) &&
           !strcmp(a->method_level, b->method_level));
```

Comment:
I think you can just compare the memory address (i.e. `a->method_level == b->method_level`) since I think we just have a couple of static strings for these.